src/test/java/org/opensearch/knn/training/VectorReaderTests.java

```diff
@@ -32,32 +31,31 @@
 
 public class VectorReaderTests extends KNNSingleNodeTestCase {
 
-    public static Logger logger = LogManager.getLogger(VectorReaderTests.class);
+    private final int DEFAULT_LATCH_TIMEOUT = 100;
+    private final String DEFAULT_INDEX_NAME = "test-index";
+    private final String DEFAULT_FIELD_NAME = "test-field";
+    private final int DEFAULT_DIMENSION = 16;
+    private final int DEFAULT_NUM_VECTORS = 100;
+    private final int DEFAULT_MAX_VECTOR_COUNT = 10000;
+    private final int DEFAULT_SEARCH_SIZE = 10;
```

Comment:
any reason for not making them static?

Reply:
Oh good question. Would it be better practice to do that?

Reply:
Yes its a better practice.